action/src/run.ts

```diff
@@ -79,14 +89,19 @@ export const run = async () => {
     });
   }
 
+  const latestVisualRegressionStatus = commitHash
+    ? await getLatestVisualRegressionStatus(commitHash)
+    : null;
+
   if (diffFileCount === 0 && newFileCount === 0) {
-    info('All visual tests passed, and no diffs found!');
+    info(VISUAL_TESTS_PASSED);
 
+    if (!commitHash) return;
     if (isRetry) {
       warning(
         'Disabling auto merge because this is a retry attempt. This is to avoid auto merging prematurely.'
       );
-      await disableAutoMerge(commitHash);
+      await disableAutoMerge(hash);
```

Comment:
```suggestion
      await disableAutoMerge(commitHash);
```

We might as well stick with commitHash since we would never call this with diffId